### PR TITLE
add basic ecommerce content

### PR DESCRIPTION
**Pull request by sushmangupta**

### CODE REVIEW

File: 1.Familiarize yourself with Shopware/1.Introduction to Shopware ecosystem/1.What is E-commerce and its terminologies.md [hunk 1]
```diff
@@ -1,5 +1,51 @@
 ---
-title: 'What is E-commerce and its terminologies'
+title: 'What is E-commerce'
 ---
 
-What is E-commerce and its terminologies CONTENT
+# What is ecommerce?
+
+**eCommerce**, short for electronic commerce, involves the buying and selling,  goods over the internet. This broad term encompasses various activities, such as online retail shopping, digital payments, internet banking, online auctions, electronic ticketing for events, and more. eCommerce leverages technologies like mobile commerce (mCommerce), electronic funds transfer (EFT), supply chain management, internet marketing, online transaction processing, inventory management systems, and automated data collection systems. By facilitating seamless and efficient transactions across the globe, eCommerce has revolutionized traditional commerce practices, making it easier for businesses and consumers to interact and conduct business online.
```
Comment (dennishoeting) [inline 1.Familiarize yourself with Shopware/1.Introduction to Shopware ecosystem/1.What is E-commerce and its terminologies.md:7]:
```suggestion
**eCommerce**, short for electronic commerce, involves the buying and selling, goods or servces over the internet. This broad term encompasses various activities, such as online retail shopping, digital payments, internet banking, online auctions, electronic ticketing for events, and more. eCommerce leverages technologies like mobile commerce (mCommerce), electronic funds transfer (EFT), supply chain management, internet marketing, online transaction processing, inventory management systems, and automated data collection systems. By facilitating seamless and efficient transactions across the globe, eCommerce has revolutionized traditional commerce practices, making it easier for businesses and consumers to interact and conduct business online.
```

File: 1.Familiarize yourself with Shopware/1.Introduction to Shopware ecosystem/1.What is E-commerce and its terminologies.md [hunk 2]
```diff
@@ -1,5 +1,51 @@
 ---
-title: 'What is E-commerce and its terminologies'
+title: 'What is E-commerce'
 ---
 
-What is E-commerce and its terminologies CONTENT
+# What is ecommerce?
```
Comment (dennishoeting) [inline 1.Familiarize yourself with Shopware/1.Introduction to Shopware ecosystem/1.What is E-commerce and its terminologies.md:5]: Idea: Include the basic pillars of commerce - it's about a catalog of products/services in categories, the customers of the shop, a checkout process, but in our idea also about an experience, so telling a story

File: 1.Familiarize yourself with Shopware/1.Introduction to Shopware ecosystem/1.What is E-commerce and its terminologies.md [hunk 3]
```diff
@@ -1,5 +1,51 @@
...
+
+## Types of eCommerce Models
+
+- B2B (Business to Business) - Transactions between businesses. Examples include manufacturers selling to wholesalers or wholesalers selling to retailers.
+Example: Alibaba's platform for businesses to buy large quantities of goods.
+
+- B2C (Business to Consumer) - Transactions between businesses and individual consumers. This is the most common form of eCommerce and includes online retail.
+Example: Amazon, where consumers can buy products directly from various brands and sellers.
+
+- C2C (Consumer to Consumer) - Transactions between consumers, often facilitated by a third-party platform. Commonly involves the sale of second-hand goods or personal services.
```
Comment (dennishoeting) [inline 1.Familiarize yourself with Shopware/1.Introduction to Shopware ecosystem/1.What is E-commerce and its terminologies.md:17]: not sure if we need C2C or C2B here (you could mention D2C also, if we wanted to). We could rather focus on B2C (the thing everybody expects) and B2B (the thing that is at least equally important) and mention that there is other models as well AND that shopware can be utilized for all of those (as you do down below)

**Review by dennishoeting — APPROVED** [review]
I like the holistic approach towards what ecommerce actually is - we should be sure to also mention what is important for us, I guess.
Alos the terminologies part is great. 

I'd approve and suggest getting into the hub - let's explore it in the system and get it feedbacked by people!

File: 1.Familiarize yourself with Shopware/1.Introduction to Shopware ecosystem/1.What is E-commerce and its terminologies.md (hunk 2 above)
Comment (sushmangupta) [reply 1.Familiarize yourself with Shopware/1.Introduction to Shopware ecosystem/1.What is E-commerce and its terminologies.md:5]: Did you mean creating a story?
I will checkout about this with you later. But for now I will add this content.

**Pull request merged**